Improve worker queues overview, other minor improvements (#1411)
astronomer/docs: astro/astro-architecture.md

description: Astro uses a multi-plane structure to help make Airflow more secure and reliable. Learn how the control plane and data plane work together on the cloud.
---

Astro is a managed service for data orchestration that is built for the cloud and powered by Apache Airflow. Astro is built with a multi-plane architecture which consists of:

- A control plane that is hosted by Astronomer.
- A data plane that can run either in your cloud or in a single-tenant account hosted by Astronomer.

Astro can be activated on Amazon Web Services (AWS), Google Cloud Platform (GCP), or Microsoft Azure. Astronomer runs Astro on managed Kubernetes services for each cloud provider, including AWS Elastic Kubernetes Service (EKS), Google Kubernetes Engine (GKE), and Azure Kubernetes Service (AKS).

> The control plane and data plane are managed by Astronomer and require no operational oversight by your team. Astro's architecture ensures that tasks are executed securely within your network and corporate boundaries, while enabling Astronomer to deliver the full promises of a managed service.

Reviewer: ```suggestion
The control plane and data plane are managed by Astronomer and require no operational oversight by your organization. The Astro architecture ensures that tasks are executed securely within your corporate network.
```

![Astro architecture overview](/img/docs/architecture-overview.png)

> For more information on Astro's architecture, contact [Astronomer support](https://cloud.astronomer.io/support).

Reviewer: ```suggestion
For more information about the Astro architecture, contact [Astronomer support](https://cloud.astronomer.io/support).
```
